app/assets/javascripts/pusherSockets.js

```diff
@@ -11,9 +11,10 @@ export const subscribeToProjectChanges = (project, callback) => {
     _,
     pusherCluster,
     pusherApiKey
-  ] = matchPusherUrl(pusherUrl) || [];
+  ] = matchPusherUrl(pusherUrl || '') || [];
```

Comment:
wouldn't it be better if you moved `|| ''` to the variable declaration above?

Reply:
Moved! Thank you, Guto :).